src/software/ai/hl/stp/tactic/chip_tactic_test.cpp

```diff
@@ -2,15 +2,292 @@
 
 #include <gtest/gtest.h>
 
+#include "software/ai/hl/stp/action/chip_action.h"
 #include "software/test_util/test_util.h"
 
-TEST(ChipTacticTest, constructor_test)
+bool compareChipActions(ChipAction *chip_action, ChipAction *expected_chip_action) {
```

Comment:
The function signature is returning a `bool` but nothing is actually returned in the function.

It could either be a `void` function that handles all the `EXPECT` statements internally, or could do all the comparisons and return a `bool` instead for the test cases to check. I have no strong preference for one or the other unless @garethellis0 does.

Reply:
Decided on using `void` and letting `EXPECT` statements be handled internally inside `compareChipActions`